Kitsune/ClubPenguin/Login.php

```diff
@@ -27,14 +27,15 @@ protected function handleLogin($socket) {
 		$password = Packet::$Data['body']['login']['pword'];
 		
 		if($penguin->database->usernameExists($username) === false) {
+			password_verify('KitsuneTimingAttack', '$2y$12$5hf7vOtBjsNJf6oZSsnyXOlcLTRblTwSg550SR0ohxVUkItmXhMM6'); //Mitigate timing attacks
```

Comment:
Correct. We do not send an error that distinguishes whether the user entered a bad password or username, and therefore we should not have a timing attack on the login that allows users to distinguish which is the issue.

Regardless of the importance of this, it would still be a security issue, since a user could use a login server timing attack to gain information that the login server is not supposed to give.